<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StalledIssueActionTitle</class>
 <widget class="QWidget" name="StalledIssueActionTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>212</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">WordWrapLabel
{
font-size: 12px;
font-weight: 600;
}

#backgroundWidget
{
border-radius: 4px;
}

/*ACTIVE*/
#backgroundWidget[disable_background=false][failed=false]
{
	background-color: #FFF5F5F5; /*colorToken.surface-1*/
}

/*NO BACKGROUND*/
#backgroundWidget[disable_background=true][failed=false]
{
	background-color: transparent;
}

/*FAILED*/
#backgroundWidget[disable_background=false][failed=true]
{
	background-color: #FFFDEBEB; /*colorToken.support-error*/
}

#backgroundWidget[disable_background=false][failed=true] WordWrapLabel
{
	color: #FAFAFB; /*colorToken.text-inverse-accent*/
}

/*DISCARDED*/
#backgroundWidget[discarded=true] WordWrapLabel
{
	color: #FAFAFB; /*colorToken.text-disabled*/
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignVCenter">
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item alignment="Qt::AlignVCenter">
       <widget class="QWidget" name="generalContainer" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="contents">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
            <property name="spacing">
             <number>8</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="titleContainer" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>8</number>
               </property>
               <property name="leftMargin">
                <number>4</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="IconLabel" name="icon">
                 <property name="minimumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignVCenter">
                <widget class="WordWrapLabel" name="titleLabel">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <family>Lato</family>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="font-size" stdset="0">
                  <string notr="true">body-2</string>
                 </property>
                 <property name="semibold" stdset="0">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="IconLabel" name="hoverIcon">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                 <property name="icon">
                  <iconset resource="../../../gui/Resources_common.qrc">
                   <normaloff>:/external_link_small_thin_outline</normaloff>:/external_link_small_thin_outline</iconset>
                 </property>
                 <property name="normal_off" stdset="0">
                  <string notr="true">icon-secondary</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="messageContainer" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QHBoxLayout" name="messageLayout">
               <property name="spacing">
                <number>5</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="IconLabel" name="iconLabel">
                 <property name="minimumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="text">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="messageLabel">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="font-size" stdset="0">
                  <string notr="true">body-2</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="extraInfoContainer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="extraInfoLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>36</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignVCenter">
       <widget class="QWidget" name="actionContainer" native="true">
        <layout class="QHBoxLayout" name="actionLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconLabel</class>
   <extends>QToolButton</extends>
   <header>TokenizableItems/IconLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WordWrapLabel</class>
   <extends>QTextEdit</extends>
   <header>WordWrapLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../gui/Resources_common.qrc"/>
 </resources>
 <connections/>
</ui>
